### PR TITLE
ocrvs-2400 Added corrector verify component

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

File: packages/client/src/views/CorrectRecord/VerifyCorrector.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        }
+      }
+    })
+
+    this.handleVerification()
+  }
+
+  isodd = (element: number, index: any, array: any) => {
+    return element % 2 == 1
+  }
```
Comment (rikukissa) [inline packages/client/src/views/CorrectRecord/VerifyCorrector.tsx:101]:
```suggestion
```

I think this is not used anywhere, right?

File: packages/client/src/views/CorrectRecord/verifyCorrector.test.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+      expect(history.location.pathname).toBe('/')
+    })
+
+    describe('when father is corrector', () => {
+      let testComponent: ReactWrapper
+      beforeEach(async () => {
+        testComponent = (
+          await createTestComponent(
+            // @ts-ignore
```
Comment (rikukissa) [inline packages/client/src/views/CorrectRecord/verifyCorrector.test.tsx:108]: Can you make this work without // @ts-ignore ?

File: packages/client/src/views/CorrectRecord/verifyCorrector.test.tsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+  describe('in case of death application renders idVerifier component', () => {
+    beforeAll(() => {
+      store.dispatch(storeApplication(deathApplication))
+    })
+
+    it('when informant is corrector', async () => {
+      const testComponent = (
+        await createTestComponent(
+          // @ts-ignore
```
Comment (rikukissa) [inline packages/client/src/views/CorrectRecord/verifyCorrector.test.tsx:177]: Same here

**Comment by sadmananik** [comment]
@rikukissa I have created a new branch and gave a PR with these fixes. Thanks.

**Pull request closed without merging**